File: Python/bolaAVA.py
# Classe Bola: Crie uma classe que modele uma bola:
# Atributos: Cor, circunferência, material
# Métodos: trocaCor e mostraCor

class bola :
    def __init__(self):
        self.__cor = "azul"
        self.__circuferencia = "80"
        self.__material = "plastico"
    def setCor (self, cor):
        self.__cor = cor
    def getCor (self):
        return print(f"a nova cor é {self.__cor}")

    

File: PHP/bolaAVA.php
<!-- Classe Bola: Crie uma classe que modele uma bola:
Atributos: Cor, circunferência, material
Métodos: trocaCor e mostraCor -->
<?php  
    class Bola{
        private $cor;
        private $circuferencia;
        private $material;
        public function trocarCor($novaCor){
        $this -> cor = $novaCor;
    }
        public function mostraCor(){
        echo "a corda bola é " + $this -> cor;
        }
    }